src/bitcoind.cpp

```diff
@@ -146,19 +142,17 @@ static bool AppInit(int argc, char* argv[])
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wdeprecated-declarations"
 #endif
-            tfm::format(std::cout, "Bitcoin server starting\n");
+            tfm::format(std::cout, PACKAGE_NAME "daemon starting\n");
```

Comment:
I don't think 'daemon' is right here when it's not started in `-daemon` mode

Reply:
This print is guarded by a DAEMON compile time check and `-daemon` run time check, so it should be all right, no?